src/QandaAtom.tsx

```diff
@@ -1,7 +1,450 @@
+                        if (!expandEventSent) {
+                            expandHandler();
+                            setExpandEventFired(true);
+                        }
+                    }
+                    setExpandStatus(!hasBeenExpanded);
+                }}
+                aria-hidden="true"
+            >
+                {!hasBeenExpanded && (
+                    <span className={iconSpacing}>
+                        <span className={plusStyling}>
+                            <SvgPlus />
+                        </span>
+                        Show
+                    </span>
+                )}
+                {hasBeenExpanded && (
+                    <span className={iconSpacing}>
+                        <span className={minusStyling}>
+                            <SvgMinus />
+                        </span>
+                        Hide
+                    </span>
+                )}
+            </span>
+        </summary>
+    );
+};
+
+const Body = ({
+    html,
+    image,
+    credit,
+}: {
+    html: string;
+    image?: string;
+    credit?: string;
+}) => (
+    <div>
+        {image && <img className={imageStyling} src={image} alt="" />}
+        <div
+            className={bodyStyling}
+            dangerouslySetInnerHTML={{
+                __html: html,
+            }}
+        />
+        {credit && <Credit credit={credit} />}
+    </div>
+);
+
+const Credit = ({ credit }: { credit: string }) => (
+    <div className={creditStyling}>
+        <span className={creditSpanStyling}>
+            <svg
+                className={creditIconStyling}
+                width="6px"
+                height="14px"
+                viewBox="0 0 6 14"
+            >
+                <path d="M4.6 12l-.4 1.4c-.7.2-1.9.6-3 .6-.7 0-1.2-.2-1.2-.9 0-.2 0-.3.1-.5l2-6.7H.7l.4-1.5 4.2-.6h.2L3 12h1.6zM4.3 2.8c-.9 0-1.4-.5-1.4-1.3C2.9.5 3.7 0 4.6 0 5.4 0 6 .5 6 1.3c0 1-.8 1.5-1.7 1.5z"></path>
+            </svg>
+        </span>
+        {' ' + credit}
+    </div>
+);
 
-export const QandaAtom = ({ id }: QandaAtomType): JSX.Element => (
-    <div data-atom-id={id}>QandaAtom</div>
+const Footer = ({
+    likeHandler,
+    dislikeHandler,
+}: {
+    likeHandler: () => void;
+    dislikeHandler: () => void;
+}) => {
+    const [showFeedback, setFeedbackVisibility] = useState(false);
+    return (
+        <footer className={footerStyling}>
+            <div hidden={showFeedback ? true : false}>
+                <div className={footerSnippet}>
+                    <div>Was this helpful?</div>
+                    <button
+                        className={buttonRound}
+                        value="like"
+                        aria-label="Yes"
+                        data-testid="like"
+                        onClick={() => {
+                            likeHandler();
+                            setFeedbackVisibility(true);
+                        }}
+                    >
+                        <svg
```

Comment:
Source icon?

Reply:
Not for thumbs at the moment

Reply:
Cool, leave as is, we can summon @guardian/design-system to see if thumbs up and thumbs down can be added in the future to refactor in.